src/pages/student/Dashboard.scss

```diff
@@ -239,7 +245,15 @@
     }
 }
 
-.dashboard-events-photo {
+.dashboard-events-photo-blob {
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+    margin-left: 150px;
+    top: -60px;
+}
+
```

Comment:
Delete this scss class since the relevant styling gets merged into the `.dashboard-blobblue-img` scss class. See Dashboard.js for why we want to merge the styling classes.
```suggestion
```